commands/debug.py

```diff
@@ -14,12 +14,24 @@
 import logging
 
 from Modules.context import Context
-from Modules.permissions import require_permission, TECHRAT, require_channel
+from Modules.permissions import require_permission, TECHRAT, RAT, require_channel
 from Modules.rat_command import command
 
 log = logging.getLogger(f"mecha.{__name__}")
 
 
+@require_permission(RAT)
+@command("ping")
+async def cmd_ping(context: Context):
+    """
+    Pongs a ping. lets see if the bots alive (command decorator testing)
+    :param context: `Context` object for the command call.
+    """
+    log.warning(f"cmd_ping triggered on channel '{context.channel}' for user "
+                f"'{context.user.nickname}'")
+    await context.reply(f"{context.user.nickname} pong!")
+
+
```

Comment:
This command was specifically coupled with main / Mechaclient. it does not belong in the commands module much less debug.

I fully intend for this command to be available in production.